apps/emqx/src/emqx_channel.erl

```diff
@@ -944,10 +937,12 @@ handle_deliver(
         clientinfo = ClientInfo
     }
 ) ->
+    % NOTE
```

Comment:
Nit, coding style: `%` -> `%%` to please the default emacs autoindentation settings